pkgs/firedragon/src.json

```
@@ -0,0 +1,15 @@
+{
+  "packageVersion": "111.0.1-1",
+  "source": {
+    "rev": "111.0.1-1",
+    "sha256": "045nisl000ll0pzir9zhrkbbkl87bsd38mygx7gz9kv6p0pppl7i"
+  },
+  "firedragon": {
+    "rev": "3a9d7188b9455bcc1fd8e713d79ff9bb3ffb32da",
+    "sha256": "BKK7ZEqf+rNK0p41dcpn8hqGRLAKxxr4kMCjcEmAeYA="
+  },
+  "firefox": {
+    "version": "111.0.1",
+    "sha512": "b16c9399a19cb1de2d865a023d54fbe71c23a363ea4d36cd58f41f64f7ad04bc1b9d8a8448943417516e17337e0ee2afd370c29a72b51b0947161f4ffab6935f"
```

```suggestion
    "hash": "sha512-sWyTmaGcsd4thloCPVT75xwjo2PqTTbNWPQfZPetBLwbnYqESJQ0F1FuFzN+DuKv03DCmnK1GwlHFh9P+raTXw=="
```

This came from: `nix hash to-sri 'sha512:b16c9399a19cb1de2d865a023d54fbe71c23a363ea4d36cd58f41f64f7ad04bc1b9d8a8448943417516e17337e0ee2afd370c29a72b51b0947161f4ffab6935f'`